client/preprocessors.py

```diff
@@ -317,17 +317,17 @@ def run(self):
             for section in toc['sections']:
                 markdown += self.compile_section(project, section, 2)
             markdown = self.fix_links(markdown)
-            output_file = os.path.join(self.output_dir, '{0}.md'.format(project.identifier))
+            output_file = os.path.join(self.output_dir, '{0}-{1}.md'.format(idx+1, project.identifier))
```

Comment:
This will work as long as `idx` is always less than 10. Would it be a good idea to check, and zero pad if needed?